frontend/pages/credits.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="../styles/credits.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Press+Start+2P&display=swap" rel="stylesheet">
+    <title>Credits</title>
+</head>
+<body>
+    <h1>Credits:</h1>
+    <h2>
+        Project Managers:
+    </h2>
+    <p>
+        Wendi Fan <br>
+        Xak Sen
+    </p>
+    <h2>
+        Developers:
+    </h2>
+    <p>
+
+    </p>
+	<button class="a" id="credits" onclick="window.location.href = 'index.html';">Return to home page</button>
```

Comment:
Where would you expect a 'return to homepage' button to live? What would it look like?